lib/src/common/model/repos.dart

```diff
@@ -459,6 +461,16 @@ class Repository {
 
   @override
   String toString() => 'Repository: $owner/$name';
+
+  static DateTime? dynamicToDateTime(dynamic time) {
+    if (time == null) {
+      return null;
+    }
+    if (time.runtimeType == int) {
```

Comment:
Can you add a note here about the different types the github api returns depending on the object for timestamps?